src/libs/ajax/DataRepo.ts

```diff
@@ -99,10 +85,32 @@ export interface Snapshot {
   cloudPlatform: 'azure' | 'gcp';
 }
 
+export interface ColumnStatisticsModel {
+  dataType: string;
+}
+
+export interface ColumnStatisticsIntOrDoubleModel extends ColumnStatisticsModel {
+  minValue: number;
+  maxValue: number;
+}
+
+export interface ColumnStatisticsTextModel extends ColumnStatisticsModel {
+  values: ColumnStatisticsTextValue[];
+}
+
+interface ColumnStatisticsTextValue {
+  value: string;
+  count: number;
```

Comment:
Having `count` is nice although we don't use it right now. I guess there's no harm in accepting it in case we want to use it in the future.

Reply:
This is more for future typing/complete types. We don't use `ColumnStatisticsTextValue` at all, count or no count. But it is one of the types this API returns so I wanted to include it.